pkg/flare/archive_docker.go

```diff
@@ -56,3 +60,35 @@ func zipDockerSelfInspect(tempDir, hostname string) error {
 	_, err = w.Write(serialized)
 	return err
 }
+
+func zipDockerPs(tempDir, hostname string) error {
+	du, err := docker.GetDockerUtil()
+	if err != nil {
+		return err
+	}
+	options := types.ContainerListOptions{All: true}
+	containerList, err := du.RawContainerList(options)
+	if err != nil {
+		return err
+	}
+
+	// Opening out file
+	f := filepath.Join(tempDir, hostname, "docker_ps.log")
+	file, err := os.Create(f)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	w := tabwriter.NewWriter(file, 20, 0, 3, ' ', 0)
```

Comment:
using `tabwriter.AlignRight` should make the output prettier

Reply:
As @arbll mentioned let's play it safe and also use the `RedactingWriter` that will strip possible sensitive info (we might have creds in the launch CMD?)

Reply:
Wasn't sure about the AlignRight :-)
Sure, didn't know the RedactingWriter was actually doing this

Thanks!